packages/core-mobile/app/services/balance/BalanceService.ts

```diff
@@ -20,6 +21,8 @@ export type BalancesForAccount = {
   error: Error | null
 }
 
+export type BalancesForXpAddress = Omit<BalancesForAccount, 'accountId'>
```

Comment:
looks like it still contains accountAddress. shouldn't this just contain the walletId?